modules/appeals_api/config/schemas/v2/200996.json

```diff
@@ -48,7 +48,7 @@
         "veteran":                            { "$ref": "#/definitions/hlrCreateVeteran" },
         "informalConferenceContact":          { "$ref": "#/definitions/hlrCreateInformalConferenceContact" },
         "informalConferenceTime":             { "$ref": "#/definitions/hlrCreateInformalConferenceTime" },
-        "representative":                     { "$ref": "#/definitions/hlrCreateRepresentative" },
+        "informalConferenceRep":              { "$ref": "#/definitions/hlrCreateRepresentative" },
```

Comment:
I've returned this key to its original name in order to get the schema in working order. Changing it caused confusion and would require unnecessary complexity in order to refactor given the number of methods on the HLR model that rely on this key. 

I will create a ticket for someone to get in here and spend time refactoring so we can move ahead with `representative` if we want.